### ISSUE TITLE
drop php 5.3-5.5 supporting

### ISSUE THREAD

**Issue by kamilsk** [octolab/Cleaner#38]
http://php.net/supported-versions.php

**Comment by falkenhawk** [comment]
Hi,
Since you are using `declare(strict_types = 1);`, you already support only PHP 7.x. 
When I tried to compose require the plugin on PHP 5.6, it resulted in a list of errors:

```
Warning: Unsupported declare 'strict_types' in [...]\vendor\octolab\cleaner\src\Plugin.php on line 3

Warning: Unsupported declare 'strict_types' in [...]\vendor\octolab\cleaner\src\Config\PluginConfig.php on line 3

Warning: Unsupported declare 'strict_types' in [...]\vendor\octolab\cleaner\src\Util\FileCleaner.php on line 3

Warning: Unsupported declare 'strict_types' in [...]\vendor\octolab\cleaner\src\Util\CleanerInterface.php on line 1

Warning: Unsupported declare 'strict_types' in [...]\vendor\octolab\cleaner\src\Util\WeightMatcher.php on line 3

Warning: Unsupported declare 'strict_types' in [...]\vendor\octolab\cleaner\src\Util\MatcherInterface.php on line 1

Warning: Unsupported declare 'strict_types' in [...]\vendor\octolab\cleaner\src\Util\CategoryNormalizer.php on line 3

Warning: Unsupported declare 'strict_types' in [...]\vendor\octolab\cleaner\src\Util\NormalizerInterface.php on line 1
Plugin installation failed, rolling back
  - Removing octolab/cleaner (dev-master)

```

Could you make your plugin compatible with at least PHP 5.6, please?

> <small>fixed by https://github.com/octolab/Cleaner/pull/39</small>

**Comment by kamilsk** [comment]
fixed by f9625224910e2dc0cab3a283fd639bba18624025
